sdk/communication/communication-rooms/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.0.0 (2023-06-14)
```

Comment:
Should be updated to: 2023-06-12, this is the date we're actually releasing

Reply:
Updated to 023-06-12